preview-site-src/elements/table.adoc

```diff
@@ -405,4 +405,94 @@ Here is a second paragraph in the same cell.
 
 And here is another paragraph.
 |*Response timeout* |Specify how long the endpoint must wait for a response (in ms). The default is *1000* ms.
-|===
\ No newline at end of file
+|===
+
+== Table Borders
+
+ref: https://docs.asciidoctor.org/asciidoc/latest/tables/borders
+
+=== frame-show
+
+[frame=show]
+|===
+|Column 1, header row |Column 2, header row |Column 3, header row
+
+|Cell in column 1, row 2
+|Cell in column 2, row 2
+|Cell in column 3, row 2
+
+|Cell in column 1, row 3
+|Cell in column 2, row 3
+|Cell in column 3, row 3
+
+|Cell in column 1, row 4
+|Cell in column 2, row 4
+|Cell in column 3, row 4
+
+|Cell in column 1, row 5
+|Cell in column 2, row 5
+|Cell in column 3, row 5
+|===
+
+=== grid-rows
+
+[grid=rows]
+|===
+|Column 1, header row |Column 2, header row |Column 3, header row
+
+|Cell in column 1, row 2
+|Cell in column 2, row 2
+|Cell in column 3, row 2
+
+|Cell in column 1, row 3
+|Cell in column 2, row 3
+|Cell in column 3, row 3
+
+|Cell in column 1, row 4
+|Cell in column 2, row 4
+|Cell in column 3, row 4
+
+|Cell in column 1, row 5
+|Cell in column 2, row 5
+|Cell in column 3, row 5
+|===
+
+=== grid-cols
+
+[grid=cols]
+|===
+|Test 1, header row |Test 2, header row |Test 3, header row
+
+|Cell in column 1, row 2
+|Cell in column 2, row 2
+|Cell in column 3, row 2
+
+|Cell in column 1, row 3
+|Cell in column 2, row 3
+|Cell in column 3, row 3
+
+|Cell in column 1, row 4
+|Cell in column 2, row 4
+|Cell in column 3, row 4
+
+|Cell in column 1, row 5
+|Cell in column 2, row 5
+|Cell in column 3, row 5
+|===
+
+=== combination of frame and grid
```

Comment:
maybe specify this as frame-show and grid-none to make it more clear

Reply:
@cheungaryk updated!